### PR TITLE
alert positioning at the top of page

### PR DESCRIPTION
**Pull request by rakeshyc**
Changes to stick Alerts at the top of the page
image

image

### CODE REVIEW

**Comment by rakeshyc** [comment]
have a doubt in the design, will have to clear it up with victoria

File: scss/product/components/_alert.scss [hunk 1]
```diff
@@ -63,3 +63,17 @@
     }
   }
 }
+//alert for displaying on top of the page.
+.pagetop-alert{
```
Comment (Pallavi-PH) [inline scss/product/components/_alert.scss:67]: Can you also add this to mdx file?

Comment (rakeshyc) [reply scss/product/components/_alert.scss:73]: added to the mdx file,

**Review by Pallavi-PH — APPROVED** [review]
LGTM

**Pull request merged**